### PR TITLE
Update flow, urls and naming conventions

### PR DESCRIPTION
**Pull request by marineb**

### CODE REVIEW

File: tests/Controller/EnMarche/AdherentControllerTest.php [hunk 1]
```diff
@@ -104,10 +104,10 @@ public function testProfileActionIsAccessibleForAdherent($profilePage, $title)
 
     public function provideProfilePage()
     {
-        yield ['/espace-adherent/mon-profil', 'Informations personnelles'];
-        yield ['/espace-adherent/mon-profil/centres-d-interet', 'Centres d\'intérêt'];
-        yield ['/espace-adherent/mon-profil/changer-mot-de-passe', 'Mot de passe'];
-        yield ['/espace-adherent/mon-profil/preferences-des-emails', 'Préférences des e-mails'];
+        yield ['/espace-adherent/mon-compte', 'Informations personnelles'];
+        yield ['/espace-adherent/mon-profil', 'Profil En Marche'];
```
Comment (HeahDude) [inline tests/Controller/EnMarche/AdherentControllerTest.php]: This line should be removed.

File: tests/Controller/EnMarche/AdherentControllerTest.php [hunk 2]
```diff
@@ -104,10 +104,10 @@ public function testProfileActionIsAccessibleForAdherent($profilePage, $title)
 
     public function provideProfilePage()
     {
-        yield ['/espace-adherent/mon-profil', 'Informations personnelles'];
-        yield ['/espace-adherent/mon-profil/centres-d-interet', 'Centres d\'intérêt'];
```
Comment (HeahDude) [inline tests/Controller/EnMarche/AdherentControllerTest.php]: Is this page removed, if not please change "profil" by "compte".

Comment (marineb) [inline tests/Controller/EnMarche/AdherentControllerTest.php]: @HeahDude but I did already. No?

File: tests/Controller/EnMarche/AdherentControllerTest.php (hunk 1 above)
Comment (marineb) [inline tests/Controller/EnMarche/AdherentControllerTest.php]: @HeahDude You mean this one? `yield ['/espace-adherent/mon-profil', 'Profil En Marche'];`

Comment (illusionOfParadise) [inline tests/Controller/EnMarche/AdherentControllerTest.php]: Yes.
I have already removed this line, but not push commit. It also corrige tests.

File: tests/Controller/EnMarche/AdherentControllerTest.php (hunk 2 above)
Comment (HeahDude) [inline tests/Controller/EnMarche/AdherentControllerTest.php]: No it's now missing.

**Pull request merged**